src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -14,8 +14,8 @@
 
 (react/defc Page
   {:render
-   (fn [{:keys [state refs this]}]
-     [:div {:style {:textAlign "center" :width "50vw"}}
+   (fn [{:keys [state props refs this]}]
+     [:div {:style {:textAlign "center"}}
       (when (:loading? @state)
         [comps/Blocker {:banner "Uploading file..."}])
```

Comment:
:width "50vw" is taken out because it was causing the width of the modal to span the width of the browser.